Documentation/ApiOverview/FormEngine/Rendering/_InputTextElement.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+namespace MyVendor\MyExtension\Backend\Form;
+
+use TYPO3\CMS\Backend\Form\Element\AbstractFormElement;
+
+class InputTextElement extends AbstractFormElement
+{
+    protected $defaultFieldWizard = [
+        'localizationStateSelector' => [
+            'renderType' => 'localizationStateSelector',
+        ],
+        'otherLanguageContent' => [
+            'renderType' => 'otherLanguageContent',
+            'after' => [
+                'localizationStateSelector',
+            ],
+        ],
+        'defaultLanguageDifferences' => [
+            'renderType' => 'defaultLanguageDifferences',
+            'after' => [
+                'otherLanguageContent',
+            ],
+        ],
+    ];
+
+    public function render()
```

Comment:
```suggestion
    public function render(): array
```

We can use covariance here for better type safety.

Reply:
No change and no response. Have you overlooked it?